apps/condo/domains/common/utils/FiltersStorage.ts

```diff
@@ -25,7 +25,17 @@ export class FiltersStorage {
     public static saveFilters (organizationId: string, tableKey: FILTER_TABLE_KEYS, filters: FiltersFromQueryType | string): void {
         const oldFilters = this.getFilters(tableKey)
 
-        localStorage.setItem(tableKey, JSON.stringify(pickBy({ ...oldFilters, [organizationId]: filters })))
+        const data2Save = pickBy({ ...oldFilters, [organizationId]: filters })
```

Comment:
`data` would be enough actually, and we don't use shortcuts like `2` instead of `To` :)

Reply:
`data2Save` replaced with `dataToSave`
In case of short `data` there is not understandable purpose of the variable